src/domain/schema.graphql

```diff
@@ -0,0 +1,152 @@
+# A workaround for debugging.
+type Debug @entity {
+	id: ID!
+	message: String!
+}
+
+enum Outcome {
+	Pass
+	Fail
+}
+
+enum Stage {
+	Open
+	Boosted
+	Overtime
```

Comment:
How to differentiate between rejected and Overtime and Resolved ..
what are the diff between each ?

Reply:
using `boostedAt`, `overtimedAt`,